pentaho-js/src/test/javascript/cdf/components/TableComponent-spec.js

```diff
@@ -196,5 +196,27 @@ define([
       dashboard.update(tableComponent);
     });
 
+    /**
+     * ## The Table Component # can update when server side is true
+     */
+    it("can update when server side is true", function(done) {
```

Comment:
This component is way too big and we need to test this following the cdf workflow. This is why we need these callThrough